### ISSUE TITLE
[rawhide] (38.20221018.91.0) kernel warning for kola tests in azure

### ISSUE THREAD

**Issue by gursewak1997** [coreos/fedora-coreos-tracker#1326]
We see multiple kola tests failing with kernel warnings in Azure.
Example test failure:
```
[2022-10-18T14:37:08.076Z] --- FAIL: coreos.tls.fetch-urls (543.19s)
[2022-10-18T14:37:08.076Z]         harness.go:1506: Found kernel warning (drivers/net/hyperv/rndis_filter.c:338 rndis_filter_receive+0x6f4/0x740 [hv_netvsc]
```
We get `WARNING: CPU: 0 PID: 580 at drivers/net/hyperv/rndis_filter.c:338 rndis_filter_receive+0x6f4/0x740 [hv_netvsc]` resulting in kernel warning after we do the [consoleCheck](https://github.com/gursewak1997/coreos-assembler/blob/main/mantle/kola/harness.go#L150).

kola-azure rawhide [pileline
](https://jenkins-fedora-coreos-pipeline.apps.ocp.fedoraproject.org/job/kola-azure/158/)
[console.txt](https://github.com/coreos/fedora-coreos-tracker/files/9843459/console.txt)
[journal.txt](https://github.com/coreos/fedora-coreos-tracker/files/9843461/journal.txt)

**Comment by dustymabe** [comment]
Looks like it could be fixed upstream by [017e425](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=017e42540639a46fdf7c7f5ee647e0b7806c9013).

**Comment by dustymabe** [comment]
Looks like this started failing on this transition:

- `kernel 6.0.0-54.fc38.x86_64 → 6.1.0-0.rc1.15.fc38.x86_64`

**Comment by gursewak1997** [comment]
Fixed by latest kernel NVR kernel-6.1.0-0.rc2.21.fc38.
The latest rawhide kola-azure [build](https://jenkins-fedora-coreos-pipeline.apps.ocp.fedoraproject.org/blue/organizations/jenkins/kola-azure/detail/kola-azure/170/pipeline) passed successfully.
